pom.xml

```diff
@@ -493,15 +493,15 @@
   </build>
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <project.http.version>1.41.7</project.http.version>
+    <project.http.version>1.41.8</project.http.version>
```

Comment:
This version depends on appengine-api 1.9.X:

https://github.com/googleapis/google-http-java-client/releases/tag/v1.41.8